### ISSUE TITLE
Validation data in Table1 

### ISSUE THREAD

**Issue by AbdulMoqeet** [clovaai/cutblur#6]
Hello! 

I have some questions regarding Table 1

You have shown the results for DIV2K and RealSR. I wanted to know whether you use Div2K validation for both DIV2K and RealSR? i.e. Trained with DIV2K and RealSR but use DIV2K validation data for testing both RealSR and DIV2K. 

Thanks.

**Comment by nmhkahn** [comment]
Hi.
For the DIV2K column, we trained the model using the DIV2K training dataset and eval on DIV2K validation set. However, for the RealSR column, we trained the model using **RealSR** dataset and eval on RealSR **test** dataset (since RealSR doesn't have validation dataset).

**Comment by AbdulMoqeet** [comment]
Thank you for the clarification.
